crates/ev-cli/src/main.rs

```diff
@@ -55,7 +56,7 @@ where
 #[clap(name = "Evervault Enclave CLI", version)]
 pub struct BaseArgs {
     /// Toggle verbose output
-    #[clap(short, long, global = true)]
+    #[clap(short, long, global = true, default_value_t = true)]
```

Comment:
Verbose default true?

Reply:
.. Dealing with merging the original CLI having a "quiet" flag where it prints ~nothing, and the enclave cli having a verbose flag where it prints ~slightly more.. Temporary solution